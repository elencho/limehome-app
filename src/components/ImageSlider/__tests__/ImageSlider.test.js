import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import ImageSlider from "../ImageSlider";

it("renders correctly", () => {
  const tree = renderer.create(<ImageSlider />).toJSON();
  expect(tree).toMatchSnapshot();
});
