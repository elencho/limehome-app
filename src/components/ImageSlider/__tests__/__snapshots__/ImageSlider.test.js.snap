// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "height": 600,
    }
  }
>
  <RCTScrollView
    horizontal={true}
    pagingEnabled={true}
    showsHorizontalScrollIndicator={false}
  >
    <View />
  </RCTScrollView>
</View>
`;
